### DISCUSSION TITLE
Why does Slidev override the default typographic quotation marks in markdown-it?

### DISCUSSION THREAD

**Discussion by rwnobrega** [slidevjs/slidev#1660] [Q&A] [answered]
I've noticed that Slidev overrides the default options of `markdown-it` for typographic quotation marks. In the Slidev code, the quotes are set to:

```javascript
quotes: '""\'\'',
```

(You can find the relevant code [here](https://github.com/slidevjs/slidev/blob/0d95a36f23bd58aae4fd4439826dbd6a03603d04/packages/slidev/node/vite/markdown.ts#L73)).

The default value in `markdown-it` is:

```javascript
quotes: '“”‘’',
```

This default value seems to be more sensible. Is there any specific reason for this override in Slidev? How can customize this choice in Slidev?

**Comment by kermanx** [comment, marked as answer]
I don't know the exact reason of this configuration. I guess maybe `"'` always looks the same on both English and Chinese context? 

To override this configuration, you can add something in the `slidev.markdown` as described in https://sli.dev/custom/config-vite#configure-internal-plugins

**Reply by rwnobrega** [reply]
Thanks! Based on your answer I managed to change the quotation marks with the following `vite.config.ts`:

```ts
import { defineConfig } from "vite";

export default defineConfig({
  //@ts-ignore
  slidev: {
    markdown: {
      markdownItSetup(md) {
        md.set({ quotes: "“”‘’" });
      },
    },
  },
});
```
